### PR TITLE
[FIX] Link 'add gateway' to fund page (RT-3285)

### PR DESCRIPTION
**Pull request by mrajvanshy**

### CODE REVIEW

File: src/jade/tabs/widgets/balances.jade [hunk 1]
```diff
@@ -83,4 +83,4 @@
                 ) {{ component | rpamount:{tiny_precision: 3} }}
 
     p(ng-show="$route.current.tabName == 'balance'")
-      a.btn.btn-cancel(href="#/trust", l10n) add gateway
+      a.btn.btn-cancel(href="#/xrp", l10n) add gateway
```
Comment (yongsoo) [inline src/jade/tabs/widgets/balances.jade]: @annatonger @mrajvanshy Is /xrp page the right page to route to?  Maybe /trust makes more sense in the other spot where we have add gateway link?

Comment (mrajvanshy) [inline src/jade/tabs/widgets/balances.jade]: @yongsoo I agree that it should link to the trust page. What do you think @annatonger?

**Comment by annatonger** [comment]
I think this PR is right, it should go to the Fund page, because it has both the easy add gateways and the manual add gateways

**Comment by yongsoo** [comment]
LGTM

**Pull request merged**